content/integrations/kubernetes.md

```diff
@@ -82,11 +83,72 @@ Since the agent is deployed as a docker container, refer to the Agent [container
 
 <%= insert_example_links%>
 
+## Kubernetes State Metrics
+
+If you are running Kubernetes >= 1.2.0, you can use the [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) project to provide additional metrics (identified by the `kubernetes_state` prefix in the metrics list below) to Datadog.
+
+To run kube-state-metrics, create a `kube-state-metrics.yaml` file using the following manifest to deploy the kube-state-metrics service:
+
+    apiVersion: extensions/v1beta1
+    kind: Deployment
+    metadata:
+      name: kube-state-metrics
+    spec:
+      replicas: 1
+      template:
+        metadata:
+          labels:
+            app: kube-state-metrics
+        spec:
+          containers:
+          - name: kube-state-metrics
+            image: gcr.io/google_containers/kube-state-metrics:v0.3.0
+            ports:
+            - name: metrics
+              containerPort: 8080
+            resources:
+              requests:
+                memory: 30Mi
+                cpu: 100m
+              limits:
+                memory: 50Mi
+                cpu: 200m
+    ---
+    apiVersion: v1
+    kind: Service
+    metadata:
+      annotations:
+        prometheus.io/scrape: 'true'
+      labels:
+        app: kube-state-metrics
+      name: kube-state-metrics
+    spec:
+      ports:
+      - name: metrics
+        port: 8080
+        targetPort: metrics
+        protocol: TCP
+      selector:
+        app: kube-state-metrics
+{:.language-yaml}
+
+Then deploy it by running:
+
+    kubectl create -f kube-state-metrics.yaml
+
+The manifest above uses Google's publicly available kube-state-metrics container. If you would like to build your own, you can do so by:
+
+1. Clone the [kube-state-metrics Github repository](https://github.com/kubernetes/kube-state-metrics)
+1. Run `make container` to build the container
+1. Run `kubectl apply -f kubernetes`
+
+If you configure your Kubernetes State Metrics service to run on a different URL or port, you can configure the Datadog Agent by setting the `kube_state_url` parameter in `conf.d/kubernetes_state.yaml`, then restarting the agent. For more information, see the [kubernetes_state.yaml.example file](https://github.com/DataDog/dd-agent/blob/master/conf.d/kubernetes_state.yaml.example)
```

Comment:
can we add a note that this should be automatically configured if you enable SD_BACKEND ?